blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,88 @@
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-Parallels.jpg)
+
+## Takeaways about Parallels
+
+In less than an hour, I was able to spin up a Windows VM on my Mac and use that environment to build a React Native for Windows app. Windows VMs allowed me to quickly experiment with adding Windows support to my existing React Native app because of the low level of overhead to get a VM up and running.
+
+## Creating a Windows VM using Microsoft Dev Box
+
+The second Windows VM option I tried was [Microsoft Dev Box](https://azure.microsoft.com/products/dev-box/). Microsoft Dev Box is a [Microsoft Azure](https://azure.microsoft.com) product that allows developers to spin up a Windows VM from most modern computers including MacOS devices.
+
+Microsoft Dev Box supports:
+
+1. Cloud-based virtual desktop infrastructure.
+1. Project-based dev box configurations which allow for code repositories and tools for your current task to be preinstalled.
+1. Region-specific dev boxes to help ensure that dev team members have a high-fidelity experience anywhere in the world.
+1. Secured cloud workstations via Microsoft Intune.
+
+Before you get started creating a VM with Microsoft Dev Box, there are a couple troubleshooting callouts I want to mention:
+
+1. You will need an Azure subscription that is not the ["free trial"](https://azure.microsoft.com/free/) subscription. You must upgrade to the ["Pay As You Go"](https://azure.microsoft.com/pricing/purchase-options/pay-as-you-go/) subscription to use Microsoft Dev Box, but you can use the $200 of free credits that you gained via the "free trial" subscription as initial payment within the “Pay As You Go” plan. If you try to create a Dev Box Definition with the "free trial" Azure subscription, you will receive a "Quota has been reached" error.
+1. You will also need a work or school Microsoft Account (MSA). Personal or guest MSAs cannot be used with the Dev Box portal at this time.
+1. Your machine will need to be managed with [Microsoft Intune](https://www.microsoft.com/security/business/Microsoft-Intune).
+
+I followed Microsoft's documentation to create a dev box using Microsoft Dev Box. I found the documentation to be straightforward to follow, and there are also several tutorial videos online from developers if you prefer to see someone walk through the steps live. You'll need to complete two steps to get your dev box up and running: [Setting Up Your Dev Box Service](https://learn.microsoft.com/azure/dev-box/quickstart-configure-dev-box-service) and [Creating a Dev Box](https://learn.microsoft.com/azure/dev-box/quickstart-create-dev-box). These steps should take about an hour to complete.
```

Comment:
Any of those mentioned videos you think are worth linking here?

Reply:
I didn't want to link any because they are just from misc youtubers; so I didn't want to explicitly promote unofficial content.